tests/integration/build_invoke/dotnet/test_build_invoke_dotnet8.py

```diff
@@ -0,0 +1,93 @@
+from tests.integration.build_invoke.build_invoke_base import BuildInvokeBase
+
+"""
+sam build does not support to build dotnetcore 3.1 templates using container, 
+here we put them in a separate file and use a dedicate codebuild project with .net 3.1 runtime to build them.
+
+For each template, it will test the following sam commands:
+1. sam init
+2. sam build --use-container (if self.use_container is False, --use-container will be omitted)
+3. (if there are event jsons), for each event json, check `sam local invoke` response is a valid json
+"""
+
+
+class BuildInvoke_dotnet8_cookiecutter_aws_sam_hello_dotnet(BuildInvokeBase.DotNetCoreExtraRerunBuildInvokeBase):
+    use_container = False
+    directory = "dotnet8/hello"
+    should_test_lint = False
+
+class BuildInvoke_dotnet8_cookiecutter_aws_sam_hello_native_aot_dotnet(BuildInvokeBase.DotNetCoreExtraRerunBuildInvokeBase):
+    use_container = False
+    directory = "dotnet8/hello-native-aot"
```

Comment:
This template is not been added to manifest JSON file

Reply:
Thanks, good catch. I'll update that.

Reply:
It's now added to the PR. My understanding is that it'll show up as 
`sam init ->  AWS Quick Start Templates -> Hello World Example -> dotnet8 -> Hello World Example using native AOT` but I'm not sure how to test it locally yet.

Reply:
Updated with testing instruction here: https://github.com/aws/aws-sam-cli-app-templates/pull/475/commits/5abf47deab40130485c3e90293619983539ff15a

Reply:
Full output of the local test is added to the PR description